thirdeye-ui/src/app/components/anomalies-create/create-anomaly-wizard/create-anomaly-wizard.component.tsx

```diff
@@ -354,28 +354,114 @@ export const CreateAnomalyWizard: FunctionComponent<CreateAnomalyWizardProps> =
             }
         };
 
-        const findClosestAppropriateTimestamp = (
-            thresholdValue: number
-        ): number | null => {
-            // This function is only useful if evaluation is defined
+        const chartDataBounds = useMemo<{
+            start: number;
+            end: number;
+        } | null>(() => {
             if (!evaluation) {
                 return null;
             }
 
             const detectionEvaluation =
                 extractDetectionEvaluation(evaluation)[0];
             const { timestamp } = detectionEvaluation.data;
+            if (timestamp.length === 0) {
+                return null;
+            }
 
-            const nextClosestTimestamp = timestamp.find(
-                (v) => v >= thresholdValue
-            );
+            return {
+                start: timestamp[0],
+                end: timestamp[timestamp.length - 1],
+            };
+        }, [evaluation]);
+
+        /** @function Ensure that the date range values are always valid and make sense
+         *  @return *boolean* indicating if the attempted correction was successful or not
```

Comment:
I follow `JSDOC` https://jsdoc.app/tags-returns.html

Change to
```
@return {boolean} indicating if the attempted correction was successful or not
```

Reply:
Updated according to the JSDOC standard